Gaijinizer patch/Map099.txt

```diff
@@ -189,18 +200,19 @@ lost interest in "humans" ...」#MTLed
 　　あのプロジェクトも凍結したかと思っていた…。
 　　安心したよ………。」
 > CONTEXT: Map099/events/1/pages/0/120/Dialogue
-*「Recently, especially, I'm crazy about remodeling and
-experiment of demon...I thought that project was also
-frozen .... I was relieved……….」#MTLed
+*「You've been quite focused on altering demons and on
+   those experiments of yours, lately...
+   I had thought that project frozen. I'm relieved...」#Ron
```

Comment:
Some rewording for flow...
>_**You have been quite focused on your demon
modification experiments lately...
I thought that project was frozen. I'm relieved...**_